packages/discord.js/src/managers/CategoryChannelChildManager.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const CachedManager = require('./CachedManager');
+const GuildChannel = require('../structures/GuildChannel');
+
+/**
+ * Manages API methods for CategoryChannels' children and stores their cache.
+ * @extends {CachedManager}
+ */
+class CategoryChannelChildManager extends CachedManager {
+  constructor(channel, iterable) {
+    super(channel.client, GuildChannel, iterable);
+    /**
+     * The category channel this manager belongs to
+     * @type {CategoryChannel}
+     */
+    this.channel = channel;
+
+    /**
+     * The guild this manager belongs to
+     * @type {Guild}
+     */
+    this.guild = channel.guild;
```

Comment:
Is it necessary to always store this? We can make it a getter.

Reply:
Yeah it's unnecessary, I don't know if the getter is even needed, what do you think?

Reply:
I think it's more a convenience, just like `message.guild` before the `guildId` days (hint: it was just `this.channel.guild`).